addons/web_editor/static/lib/summernote/src/js/Renderer.js

```diff
@@ -420,6 +420,11 @@ define([
           event: 'resize',
           value: '1'
         });
+        var threequartersButton = tplButton('<span class="note-fontsize-10">75%</span>', {
+          title: lang.image.resizeThreeQuarters,
+          event: 'resize',
+          value: '0.75'
+        });
```

Comment:
Sorry but you modify the lib directly here... I think you can drop this commit entirely while waiting for the new editor :/